net-vpn/i2pd/i2pd-2.46.1.ebuild

```diff
@@ -11,7 +11,7 @@ SRC_URI="https://github.com/PurpleI2P/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
 
 LICENSE="BSD"
 SLOT="0"
-KEYWORDS="~amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+KEYWORDS="amd64 ~arm ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
```

Comment:
Hey, we don't let users stabilize packages as it follows a "strict" process we can't be sure you've followed. Please file a stabilization bug at our bugzilla, I can help do that if needed.

Reply:
ah, sorry. Removed the stabilization commit, will create a bug